### ISSUE TITLE
Loss Behavior in LLM-based TTS Training

### ISSUE THREAD

**Issue by gevmin94** [FunAudioLLM/CosyVoice#1587]
<img width="963" height="454" alt="Image" src="https://github.com/user-attachments/assets/82e1da12-c73c-4dd3-bedf-cfe5112fa7c3" />

Hi,

We’re currently experimenting with an LLM-based TTS system and are adapting the LLM component using Qwen 3-0.6B. One behavior we’ve noticed is that the loss initially rises during the first few thousand steps before it starts to decrease.

We’ve tried different hyperparameters, optimizers, and schedulers, but this pattern persists. Since I know you’ve worked on similar setups, I’d love to hear about your experience with this issue.
